### ISSUE TITLE
[Screen Reader – BotKit – Docs]: Focus is moving twice for Image links in Scan Mode.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#13]
**User Experience:**
Users who rely on screen readers will affect to navigate if focus is moving twice for the Images links in the page.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Screen Reader: Narrator

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Enable Narrator from desktop.
Step 3: Enable Scan Mode (Caps Lock + Spacebar)
Step 3: Navigate to “Docs” link and select it.
Step 4: Navigate to “Introduction” link and select it.
Step 5: Navigate to “npm v0.7.4, dependencies up to date and License MIT” Image links in Scan Mode using Up and down arrow keys.
Step 6: Observe whether focus is moving twice for the Image links.

**Actual Result:**
Focus is moving twice for “npm v0.7.4, dependencies up to date and License MIT” Image links in Scan Mode using up and down arrow keys.

**Expected Result:**
Focus should move only one time for “npm v0.7.4, dependencies up to date and License MIT” Image links in Scan Mode using up and down arrow keys.

[Usable_BotKit_Docs.zip](https://github.com/howdyai/botkit-website/files/2976746/Usable_BotKit_Docs.zip)

**Comment by benbrown** [comment]
I cannot replicate this.  What would typically cause this?  It is a simple A tag wrapped around an image.

**Comment by v-gasikk** [comment]
Issue is Not Fixed.
When user navigating through Scan mode, focus is moving twice for the badges and for every element of the badge starting it is narrating header element with respect to the badge.

**Comment by benbrown** [comment]
@v-gasikk @awalia13 I do not understand why this is happening -- should I add an aria-hidden tag to the image inside the link?  This is a normal link wrapped around an image.

**Comment by benbrown** [comment]
Add aria-hidden to image

**Comment by benbrown** [comment]
This should be fixed.

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug.
